index.js

```diff
@@ -66,51 +133,61 @@ const main = async () => {
+  }
+
+  const ignoredCounts = countIgnoredFindings(ignoredFindings)
   const counts = findings.imageScanFindings.findingSeverityCounts
   const critical = counts.CRITICAL || 0
   const high = counts.HIGH || 0
   const medium = counts.MEDIUM || 0
   const low = counts.LOW || 0
   const informational = counts.INFORMATIONAL || 0
   const indeterminate = counts.UNDEFINED || 0
+  const ignored = ignoreList.length
   const total = critical + high + medium + low + informational + indeterminate
   core.setOutput('critical', critical.toString())
   core.setOutput('high', high.toString())
   core.setOutput('medium', medium.toString())
   core.setOutput('low', low.toString())
   core.setOutput('informational', informational.toString())
   core.setOutput('undefined', indeterminate.toString())
+  core.setOutput('ignored', ignored.toString())
   core.setOutput('total', total.toString())
   console.log('Vulnerabilities found:')
-  console.log(`${critical.toString().padStart(3, ' ')} Critical`)
-  console.log(`${high.toString().padStart(3, ' ')} High`)
-  console.log(`${medium.toString().padStart(3, ' ')} Medium`)
-  console.log(`${low.toString().padStart(3, ' ')} Low`)
-  console.log(`${informational.toString().padStart(3, ' ')} Informational`)
+  console.log(`${critical.toString().padStart(3, ' ')} Critical ${getCount('critical', ignoredCounts)}`)
+  console.log(`${high.toString().padStart(3, ' ')} High ${getCount('high', ignoredCounts)}`)
+  console.log(`${medium.toString().padStart(3, ' ')} Medium ${getCount('medium', ignoredCounts)}`)
+  console.log(`${low.toString().padStart(3, ' ')} Low ${getCount('low', ignoredCounts)}`)
+  console.log(`${informational.toString().padStart(3, ' ')} Informational ${getCount('informational', ignoredCounts)}`)
   console.log(`${indeterminate.toString().padStart(3, ' ')} Undefined`)
   console.log('=================')
-  console.log(`${total.toString().padStart(3, ' ')} Total`)
+  console.log(`${total.toString().padStart(3, ' ')} Total ${getCount('total', ignoredCounts)}`)
 
   const numFailingVulns =
-    failThreshold === 'informational' ? total
-      : failThreshold === 'low' ? critical + high + medium + low
-        : failThreshold === 'medium' ? critical + high + medium
-          : failThreshold === 'high' ? critical + high
-            : /* failThreshold === 'critical' ? */ critical
+    failThreshold === 'informational' ? total - ignoredCounts.informational
+      : failThreshold === 'low' ? critical + high + medium + low - ignoredCounts.low
+        : failThreshold === 'medium' ? critical + high + medium - ignoredCounts.medium
+          : failThreshold === 'high' ? critical + high - ignoredCounts.high
+            : /* failThreshold === 'critical' ? */ critical - ignoredCounts.critical
 
   if (numFailingVulns > 0) {
     throw new Error(`Detected ${numFailingVulns} vulnerabilities with severity >= ${failThreshold} (the currently configured fail_threshold).`)
   }
 }
 
-(async function () {
+;(async function () {
```

Comment:
Is this a stylistic thing or for technical reasons? Just curious.

Reply:
Prettier 😏